plugins/modules/database/misc/odbc.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, John Westcott <john.westcott.iv@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: odbc
+author: "John Westcott IV (@john-westcott-iv)"
+version_added: "1.0.0"
+short_description: Execute SQL via ODBC
+description:
+    - Read/Write info via ODBC drivers.
+options:
+    dsn:
+      description:
+        - The connection string passed into ODBC.
+      required: yes
+      type: str
+    query:
+      description:
+        - The SQL query to perform.
+      required: yes
+      type: str
+    params:
+      description:
+        - Parameters to pass to the SQL query.
+      type: list
+      elements: str
+
+requirements:
+  - "python >= 2.6"
+  - "pyodbc"
+
+notes:
+  - "Like the command module, this module always returns changed = yes whether or not the query would change the database."
+  - "To alter this behavior you can use C(changed_when): [yes or no]."
+  - "For details about return values (description and row_count) see U(https://github.com/mkleehammer/pyodbc/wiki/Cursor)."
+'''
+
+EXAMPLES = '''
+- name: Set some values in the test db
+  odbc:
```

Comment:
Sorry, I missed this one before:
```suggestion
  community.general.odbc:
```

Reply:
Interesting. In general do we recommend people to fully qualify their modules with their name? Or do we push people towards loading them with the collections keyword:
```
collections:
  - community.general
```

Reply:
The docs team decided that we should be using FQCNs everywhere in the docs. Then it is clear what is meant, the examples will always work and not depend on the right `collection:` be used, and people understanding `collections:` will have no problem adjusting the examples.